src/views/Predictions/icons/PocketWatch.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import { Svg, SvgProps } from '@pancakeswap-libs/uikit'
+
+const PocketWatch: React.FC<SvgProps> = (props) => (
```

Comment:
This one in the uikit, please remove it

Reply:
All the icons in this folder should probably go into the uikit
- We don't know if they will be used later
- There is no way for a dev to easily check what this icon is
With the icons in this folder, we will likely end up having some duplicated content